st2common/st2common/models/api/rbac.py

```diff
@@ -31,7 +34,7 @@
 
 
 class RoleAPI(BaseAPI):
-    model = PackDB
```

Comment:
This code path and model attribute is only used when creating objects through the API and that's not what happens with RBAC (we have a read-only API) so this code path was never exercised.